.github/workflows/CD-publish-deps-image.yml

```diff
@@ -74,7 +74,10 @@ jobs:
             docker push ghcr.io/stjude/ppfull:$TAG
```

Comment:
@aacic The CI should run `npm run test:integration --workspace=server`  from the pp root, after the build and before docker push/updating `update_deps_version.sh`. The reason this step was not automated before was manual testing is required per instructions in https://github.com/stjude/proteinpaint/wiki/Buiding-a-deps-image. If we automate the last steps to update the `ARG VERSION=...` in that wiki, we should also automate running CI tests before publishing the new deps image and updating versions.